Build Issues with Vitruv-CLI

On my machine (Arch Linux, JDK 21, main-Branch), building Vitruv-CLI fails with the following errors, due to outdated Vitruvius dependencies:

```bash
$ ./mvnw clean verify -U 
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO] 
[INFO] Vitruv CLI                                                         [pom]
[INFO] Vitruv command line interface                                      [jar]
[INFO] p2 Dependency Wrappers                                             [pom]
[INFO] p2 Dependency Wrapper EMF Utils                                    [jar]
[INFO] 
[INFO] -------------------< tools.vitruv:tools.vitruv.cli >--------------------
[INFO] Building Vitruv CLI 3.2.0-SNAPSHOT                                 [1/4]
[INFO]   from pom.xml
[INFO] --------------------------------[ pom ]---------------------------------
[INFO] 
[INFO] --- clean:3.5.0:clean (default-clean) @ tools.vitruv.cli ---
[INFO] 
[INFO] -----------------< tools.vitruv:tools.vitruv.cli.cli >------------------
[INFO] Building Vitruv command line interface 3.2.0-SNAPSHOT              [2/4]
[INFO]   from cli/pom.xml
[INFO] --------------------------------[ jar ]---------------------------------
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.framework.views/3.2.0-SNAPSHOT/maven-metadata.xml
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.framework.views/3.2.0-SNAPSHOT/maven-metadata.xml
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.framework.views/3.2.0-SNAPSHOT/tools.vitruv.framework.views-3.2.0-SNAPSHOT.pom
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.framework.views/3.2.0-SNAPSHOT/tools.vitruv.framework.views-3.2.0-SNAPSHOT.pom
[WARNING] The POM for tools.vitruv:tools.vitruv.framework.views:jar:3.2.0-SNAPSHOT is missing, no dependency information available
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.framework.vsum/3.2.0-SNAPSHOT/maven-metadata.xml
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.framework.vsum/3.2.0-SNAPSHOT/maven-metadata.xml
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.framework.vsum/3.2.0-SNAPSHOT/tools.vitruv.framework.vsum-3.2.0-SNAPSHOT.pom
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.framework.vsum/3.2.0-SNAPSHOT/tools.vitruv.framework.vsum-3.2.0-SNAPSHOT.pom
[WARNING] The POM for tools.vitruv:tools.vitruv.framework.vsum:jar:3.2.0-SNAPSHOT is missing, no dependency information available
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.change.utils/3.2.0-SNAPSHOT/maven-metadata.xml
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.change.utils/3.2.0-SNAPSHOT/maven-metadata.xml
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.change.utils/3.2.0-SNAPSHOT/tools.vitruv.change.utils-3.2.0-SNAPSHOT.pom
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.change.utils/3.2.0-SNAPSHOT/tools.vitruv.change.utils-3.2.0-SNAPSHOT.pom
[WARNING] The POM for tools.vitruv:tools.vitruv.change.utils:jar:3.2.0-SNAPSHOT is missing, no dependency information available
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.change.atomic/3.2.0-SNAPSHOT/maven-metadata.xml
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.change.atomic/3.2.0-SNAPSHOT/maven-metadata.xml
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.change.atomic/3.2.0-SNAPSHOT/tools.vitruv.change.atomic-3.2.0-SNAPSHOT.pom
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.change.atomic/3.2.0-SNAPSHOT/tools.vitruv.change.atomic-3.2.0-SNAPSHOT.pom
[WARNING] The POM for tools.vitruv:tools.vitruv.change.atomic:jar:3.2.0-SNAPSHOT is missing, no dependency information available
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.change.composite/3.2.0-SNAPSHOT/maven-metadata.xml
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.change.composite/3.2.0-SNAPSHOT/maven-metadata.xml
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.change.composite/3.2.0-SNAPSHOT/tools.vitruv.change.composite-3.2.0-SNAPSHOT.pom
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.change.composite/3.2.0-SNAPSHOT/tools.vitruv.change.composite-3.2.0-SNAPSHOT.pom
[WARNING] The POM for tools.vitruv:tools.vitruv.change.composite:jar:3.2.0-SNAPSHOT is missing, no dependency information available
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.change.correspondence/3.2.0-SNAPSHOT/maven-metadata.xml
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.change.correspondence/3.2.0-SNAPSHOT/maven-metadata.xml
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.change.correspondence/3.2.0-SNAPSHOT/tools.vitruv.change.correspondence-3.2.0-SNAPSHOT.pom
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.change.correspondence/3.2.0-SNAPSHOT/tools.vitruv.change.correspondence-3.2.0-SNAPSHOT.pom
[WARNING] The POM for tools.vitruv:tools.vitruv.change.correspondence:jar:3.2.0-SNAPSHOT is missing, no dependency information available
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.change.interaction/3.2.0-SNAPSHOT/maven-metadata.xml
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.change.interaction/3.2.0-SNAPSHOT/maven-metadata.xml
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.change.interaction/3.2.0-SNAPSHOT/tools.vitruv.change.interaction-3.2.0-SNAPSHOT.pom
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.change.interaction/3.2.0-SNAPSHOT/tools.vitruv.change.interaction-3.2.0-SNAPSHOT.pom
[WARNING] The POM for tools.vitruv:tools.vitruv.change.interaction:jar:3.2.0-SNAPSHOT is missing, no dependency information available
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.change.propagation/3.2.0-SNAPSHOT/maven-metadata.xml
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.change.propagation/3.2.0-SNAPSHOT/maven-metadata.xml
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.change.propagation/3.2.0-SNAPSHOT/tools.vitruv.change.propagation-3.2.0-SNAPSHOT.pom
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.change.propagation/3.2.0-SNAPSHOT/tools.vitruv.change.propagation-3.2.0-SNAPSHOT.pom
[WARNING] The POM for tools.vitruv:tools.vitruv.change.propagation:jar:3.2.0-SNAPSHOT is missing, no dependency information available
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.change.interaction.model/3.2.0-SNAPSHOT/maven-metadata.xml
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.change.interaction.model/3.2.0-SNAPSHOT/maven-metadata.xml
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.change.interaction.model/3.2.0-SNAPSHOT/tools.vitruv.change.interaction.model-3.2.0-SNAPSHOT.pom
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.change.interaction.model/3.2.0-SNAPSHOT/tools.vitruv.change.interaction.model-3.2.0-SNAPSHOT.pom
[WARNING] The POM for tools.vitruv:tools.vitruv.change.interaction.model:jar:3.2.0-SNAPSHOT is missing, no dependency information available
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.change.testutils.integration/3.2.0-SNAPSHOT/maven-metadata.xml
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.change.testutils.integration/3.2.0-SNAPSHOT/maven-metadata.xml
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.change.testutils.integration/3.2.0-SNAPSHOT/tools.vitruv.change.testutils.integration-3.2.0-SNAPSHOT.pom
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.change.testutils.integration/3.2.0-SNAPSHOT/tools.vitruv.change.testutils.integration-3.2.0-SNAPSHOT.pom
[WARNING] The POM for tools.vitruv:tools.vitruv.change.testutils.integration:jar:3.2.0-SNAPSHOT is missing, no dependency information available
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.server.remote/3.2.0-SNAPSHOT/maven-metadata.xml
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.server.remote/3.2.0-SNAPSHOT/maven-metadata.xml
Progress (1): 1.2 kB
                    
Downloaded from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.server.remote/3.2.0-SNAPSHOT/maven-metadata.xml (1.2 kB at 2.1 kB/s)
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.server/3.2.0-SNAPSHOT/maven-metadata.xml
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.server/3.2.0-SNAPSHOT/maven-metadata.xml
Progress (1): 791 B
                   
Downloaded from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.server/3.2.0-SNAPSHOT/maven-metadata.xml (791 B at 1.5 kB/s)
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.server.p2wrappers.emfutils/3.2.0-SNAPSHOT/maven-metadata.xml
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.server.p2wrappers.emfutils/3.2.0-SNAPSHOT/maven-metadata.xml
Progress (1): 2.0 kB
                    
Downloaded from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.server.p2wrappers.emfutils/3.2.0-SNAPSHOT/maven-metadata.xml (2.0 kB at 3.9 kB/s)
Downloading from central: https://repo1.maven.org/maven2/org/eclipse/platform/org.eclipse.core.resources/maven-metadata.xml
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/eclipse/platform/org.eclipse.core.resources/maven-metadata.xml
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/org/eclipse/platform/org.eclipse.core.resources/maven-metadata.xml
Progress (1): 1.4 kB
                    
Downloaded from central: https://repo1.maven.org/maven2/org/eclipse/platform/org.eclipse.core.resources/maven-metadata.xml (1.4 kB at 14 kB/s)
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/org/eclipse/platform/org.eclipse.core.expressions/maven-metadata.xml
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/eclipse/platform/org.eclipse.core.expressions/maven-metadata.xml
Downloading from central: https://repo1.maven.org/maven2/org/eclipse/platform/org.eclipse.core.expressions/maven-metadata.xml
Progress (1): 1.0 kB
                    
Downloaded from central: https://repo1.maven.org/maven2/org/eclipse/platform/org.eclipse.core.expressions/maven-metadata.xml (1.0 kB at 46 kB/s)
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/org/eclipse/platform/org.eclipse.core.filesystem/maven-metadata.xml
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/eclipse/platform/org.eclipse.core.filesystem/maven-metadata.xml
Downloading from central: https://repo1.maven.org/maven2/org/eclipse/platform/org.eclipse.core.filesystem/maven-metadata.xml
Progress (1): 1.1 kB
                    
Downloaded from central: https://repo1.maven.org/maven2/org/eclipse/platform/org.eclipse.core.filesystem/maven-metadata.xml (1.1 kB at 47 kB/s)
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/org/eclipse/platform/org.eclipse.osgi/maven-metadata.xml
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/eclipse/platform/org.eclipse.osgi/maven-metadata.xml
Downloading from central: https://repo1.maven.org/maven2/org/eclipse/platform/org.eclipse.osgi/maven-metadata.xml
Progress (1): 1.5 kB
                    
Downloaded from central: https://repo1.maven.org/maven2/org/eclipse/platform/org.eclipse.osgi/maven-metadata.xml (1.5 kB at 59 kB/s)
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/eclipse/jdt/org.eclipse.jdt.core/maven-metadata.xml
Downloading from central: https://repo1.maven.org/maven2/org/eclipse/jdt/org.eclipse.jdt.core/maven-metadata.xml
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/org/eclipse/jdt/org.eclipse.jdt.core/maven-metadata.xml
Progress (1): 1.6 kB
                    
Downloaded from central: https://repo1.maven.org/maven2/org/eclipse/jdt/org.eclipse.jdt.core/maven-metadata.xml (1.6 kB at 65 kB/s)
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/org/eclipse/platform/org.eclipse.text/maven-metadata.xml
Downloading from central: https://repo1.maven.org/maven2/org/eclipse/platform/org.eclipse.text/maven-metadata.xml
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/eclipse/platform/org.eclipse.text/maven-metadata.xml
Progress (1): 1.1 kB
                    
Downloaded from central: https://repo1.maven.org/maven2/org/eclipse/platform/org.eclipse.text/maven-metadata.xml (1.1 kB at 44 kB/s)
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/org/eclipse/platform/org.eclipse.core.commands/maven-metadata.xml
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/eclipse/platform/org.eclipse.core.commands/maven-metadata.xml
Downloading from central: https://repo1.maven.org/maven2/org/eclipse/platform/org.eclipse.core.commands/maven-metadata.xml
Progress (1): 1.1 kB
                    
Downloaded from central: https://repo1.maven.org/maven2/org/eclipse/platform/org.eclipse.core.commands/maven-metadata.xml (1.1 kB at 40 kB/s)
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/eclipse/platform/org.eclipse.equinox.preferences/maven-metadata.xml
Downloading from central: https://repo1.maven.org/maven2/org/eclipse/platform/org.eclipse.equinox.preferences/maven-metadata.xml
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/org/eclipse/platform/org.eclipse.equinox.preferences/maven-metadata.xml
Progress (1): 1.2 kB
                    
Downloaded from central: https://repo1.maven.org/maven2/org/eclipse/platform/org.eclipse.equinox.preferences/maven-metadata.xml (1.2 kB at 46 kB/s)
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/org/osgi/org.osgi.service.prefs/maven-metadata.xml
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/osgi/org.osgi.service.prefs/maven-metadata.xml
Downloading from central: https://repo1.maven.org/maven2/org/osgi/org.osgi.service.prefs/maven-metadata.xml
Progress (1): 389 B
                   
Downloaded from central: https://repo1.maven.org/maven2/org/osgi/org.osgi.service.prefs/maven-metadata.xml (389 B at 18 kB/s)
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/eclipse/jdt/org.eclipse.jdt.launching/maven-metadata.xml
Downloading from central: https://repo1.maven.org/maven2/org/eclipse/jdt/org.eclipse.jdt.launching/maven-metadata.xml
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/org/eclipse/jdt/org.eclipse.jdt.launching/maven-metadata.xml
Progress (1): 1.5 kB
                    
Downloaded from central: https://repo1.maven.org/maven2/org/eclipse/jdt/org.eclipse.jdt.launching/maven-metadata.xml (1.5 kB at 66 kB/s)
Downloading from central: https://repo1.maven.org/maven2/org/eclipse/platform/org.eclipse.debug.core/maven-metadata.xml
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/org/eclipse/platform/org.eclipse.debug.core/maven-metadata.xml
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/eclipse/platform/org.eclipse.debug.core/maven-metadata.xml
Progress (1): 1.4 kB
                    
Downloaded from central: https://repo1.maven.org/maven2/org/eclipse/platform/org.eclipse.debug.core/maven-metadata.xml (1.4 kB at 61 kB/s)
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/eclipse/platform/org.eclipse.core.variables/maven-metadata.xml
Downloading from central: https://repo1.maven.org/maven2/org/eclipse/platform/org.eclipse.core.variables/maven-metadata.xml
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/org/eclipse/platform/org.eclipse.core.variables/maven-metadata.xml
Progress (1): 945 B
                   
Downloaded from central: https://repo1.maven.org/maven2/org/eclipse/platform/org.eclipse.core.variables/maven-metadata.xml (945 B at 43 kB/s)
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/org/eclipse/jdt/org.eclipse.jdt.debug/maven-metadata.xml
Downloading from central: https://repo1.maven.org/maven2/org/eclipse/jdt/org.eclipse.jdt.debug/maven-metadata.xml
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/eclipse/jdt/org.eclipse.jdt.debug/maven-metadata.xml
Progress (1): 1.4 kB
                    
Downloaded from central: https://repo1.maven.org/maven2/org/eclipse/jdt/org.eclipse.jdt.debug/maven-metadata.xml (1.4 kB at 48 kB/s)
Downloading from central: https://repo1.maven.org/maven2/org/eclipse/emf/org.eclipse.emf.codegen/maven-metadata.xml
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/eclipse/emf/org.eclipse.emf.codegen/maven-metadata.xml
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/org/eclipse/emf/org.eclipse.emf.codegen/maven-metadata.xml
Progress (1): 1.1 kB
                    
Downloaded from central: https://repo1.maven.org/maven2/org/eclipse/emf/org.eclipse.emf.codegen/maven-metadata.xml (1.1 kB at 48 kB/s)
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.framework.p2wrappers.emfutils/3.2.0-SNAPSHOT/maven-metadata.xml
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.framework.p2wrappers.emfutils/3.2.0-SNAPSHOT/maven-metadata.xml
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.framework.p2wrappers.emfutils/3.2.0-SNAPSHOT/tools.vitruv.framework.p2wrappers.emfutils-3.2.0-SNAPSHOT.pom
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.framework.p2wrappers.emfutils/3.2.0-SNAPSHOT/tools.vitruv.framework.p2wrappers.emfutils-3.2.0-SNAPSHOT.pom
[WARNING] The POM for tools.vitruv:tools.vitruv.framework.p2wrappers.emfutils:jar:3.2.0-SNAPSHOT is missing, no dependency information available
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.framework.views/3.2.0-SNAPSHOT/tools.vitruv.framework.views-3.2.0-SNAPSHOT.jar
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.framework.vsum/3.2.0-SNAPSHOT/tools.vitruv.framework.vsum-3.2.0-SNAPSHOT.jar
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.change.utils/3.2.0-SNAPSHOT/tools.vitruv.change.utils-3.2.0-SNAPSHOT.jar
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.change.correspondence/3.2.0-SNAPSHOT/tools.vitruv.change.correspondence-3.2.0-SNAPSHOT.jar
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.change.interaction/3.2.0-SNAPSHOT/tools.vitruv.change.interaction-3.2.0-SNAPSHOT.jar
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.change.propagation/3.2.0-SNAPSHOT/tools.vitruv.change.propagation-3.2.0-SNAPSHOT.jar
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.change.composite/3.2.0-SNAPSHOT/tools.vitruv.change.composite-3.2.0-SNAPSHOT.jar
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.change.interaction.model/3.2.0-SNAPSHOT/tools.vitruv.change.interaction.model-3.2.0-SNAPSHOT.jar
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.change.atomic/3.2.0-SNAPSHOT/tools.vitruv.change.atomic-3.2.0-SNAPSHOT.jar
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.change.testutils.integration/3.2.0-SNAPSHOT/tools.vitruv.change.testutils.integration-3.2.0-SNAPSHOT.jar
Downloading from ossrh-snapshots: https://oss.sonatype.org/content/repositories/snapshots/tools/vitruv/tools.vitruv.framework.p2wrappers.emfutils/3.2.0-SNAPSHOT/tools.vitruv.framework.p2wrappers.emfutils-3.2.0-SNAPSHOT.jar
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.framework.views/3.2.0-SNAPSHOT/tools.vitruv.framework.views-3.2.0-SNAPSHOT.jar
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.framework.vsum/3.2.0-SNAPSHOT/tools.vitruv.framework.vsum-3.2.0-SNAPSHOT.jar
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.change.utils/3.2.0-SNAPSHOT/tools.vitruv.change.utils-3.2.0-SNAPSHOT.jar
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.change.correspondence/3.2.0-SNAPSHOT/tools.vitruv.change.correspondence-3.2.0-SNAPSHOT.jar
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.change.interaction/3.2.0-SNAPSHOT/tools.vitruv.change.interaction-3.2.0-SNAPSHOT.jar
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.change.propagation/3.2.0-SNAPSHOT/tools.vitruv.change.propagation-3.2.0-SNAPSHOT.jar
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.change.composite/3.2.0-SNAPSHOT/tools.vitruv.change.composite-3.2.0-SNAPSHOT.jar
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.change.interaction.model/3.2.0-SNAPSHOT/tools.vitruv.change.interaction.model-3.2.0-SNAPSHOT.jar
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.change.atomic/3.2.0-SNAPSHOT/tools.vitruv.change.atomic-3.2.0-SNAPSHOT.jar
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.change.testutils.integration/3.2.0-SNAPSHOT/tools.vitruv.change.testutils.integration-3.2.0-SNAPSHOT.jar
Downloading from central-portal-snapshots: https://central.sonatype.com/repository/maven-snapshots/tools/vitruv/tools.vitruv.framework.p2wrappers.emfutils/3.2.0-SNAPSHOT/tools.vitruv.framework.p2wrappers.emfutils-3.2.0-SNAPSHOT.jar
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Vitruv CLI 3.2.0-SNAPSHOT:
[INFO] 
[INFO] Vitruv CLI ......................................... SUCCESS [  0.043 s]
[INFO] Vitruv command line interface ...................... FAILURE [ 12.289 s]
[INFO] p2 Dependency Wrappers ............................. SKIPPED
[INFO] p2 Dependency Wrapper EMF Utils .................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  12.500 s
[INFO] Finished at: 2025-11-19T10:32:30+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project tools.vitruv.cli.cli: Could not resolve dependencies for project tools.vitruv:tools.vitruv.cli.cli:jar:3.2.0-SNAPSHOT
[ERROR] dependency: tools.vitruv:tools.vitruv.framework.views:jar:3.2.0-SNAPSHOT (compile)
[ERROR] 	Could not find artifact tools.vitruv:tools.vitruv.framework.views:jar:3.2.0-SNAPSHOT in ossrh-snapshots (https://oss.sonatype.org/content/repositories/snapshots)
[ERROR] 	Could not find artifact tools.vitruv:tools.vitruv.framework.views:jar:3.2.0-SNAPSHOT in central-portal-snapshots (https://central.sonatype.com/repository/maven-snapshots/)
[ERROR] dependency: tools.vitruv:tools.vitruv.framework.vsum:jar:3.2.0-SNAPSHOT (compile)
[ERROR] 	Could not find artifact tools.vitruv:tools.vitruv.framework.vsum:jar:3.2.0-SNAPSHOT in ossrh-snapshots (https://oss.sonatype.org/content/repositories/snapshots)
[ERROR] 	Could not find artifact tools.vitruv:tools.vitruv.framework.vsum:jar:3.2.0-SNAPSHOT in central-portal-snapshots (https://central.sonatype.com/repository/maven-snapshots/)
[ERROR] dependency: tools.vitruv:tools.vitruv.change.utils:jar:3.2.0-SNAPSHOT (compile)
[ERROR] 	Could not find artifact tools.vitruv:tools.vitruv.change.utils:jar:3.2.0-SNAPSHOT in ossrh-snapshots (https://oss.sonatype.org/content/repositories/snapshots)
[ERROR] 	Could not find artifact tools.vitruv:tools.vitruv.change.utils:jar:3.2.0-SNAPSHOT in central-portal-snapshots (https://central.sonatype.com/repository/maven-snapshots/)
[ERROR] dependency: tools.vitruv:tools.vitruv.change.correspondence:jar:3.2.0-SNAPSHOT (compile)
[ERROR] 	Could not find artifact tools.vitruv:tools.vitruv.change.correspondence:jar:3.2.0-SNAPSHOT in ossrh-snapshots (https://oss.sonatype.org/content/repositories/snapshots)
[ERROR] 	Could not find artifact tools.vitruv:tools.vitruv.change.correspondence:jar:3.2.0-SNAPSHOT in central-portal-snapshots (https://central.sonatype.com/repository/maven-snapshots/)
[ERROR] dependency: tools.vitruv:tools.vitruv.change.interaction:jar:3.2.0-SNAPSHOT (compile)
[ERROR] 	Could not find artifact tools.vitruv:tools.vitruv.change.interaction:jar:3.2.0-SNAPSHOT in ossrh-snapshots (https://oss.sonatype.org/content/repositories/snapshots)
[ERROR] 	Could not find artifact tools.vitruv:tools.vitruv.change.interaction:jar:3.2.0-SNAPSHOT in central-portal-snapshots (https://central.sonatype.com/repository/maven-snapshots/)
[ERROR] dependency: tools.vitruv:tools.vitruv.change.propagation:jar:3.2.0-SNAPSHOT (compile)
[ERROR] 	Could not find artifact tools.vitruv:tools.vitruv.change.propagation:jar:3.2.0-SNAPSHOT in ossrh-snapshots (https://oss.sonatype.org/content/repositories/snapshots)
[ERROR] 	Could not find artifact tools.vitruv:tools.vitruv.change.propagation:jar:3.2.0-SNAPSHOT in central-portal-snapshots (https://central.sonatype.com/repository/maven-snapshots/)
[ERROR] dependency: tools.vitruv:tools.vitruv.change.composite:jar:3.2.0-SNAPSHOT (compile)
[ERROR] 	Could not find artifact tools.vitruv:tools.vitruv.change.composite:jar:3.2.0-SNAPSHOT in ossrh-snapshots (https://oss.sonatype.org/content/repositories/snapshots)
[ERROR] 	Could not find artifact tools.vitruv:tools.vitruv.change.composite:jar:3.2.0-SNAPSHOT in central-portal-snapshots (https://central.sonatype.com/repository/maven-snapshots/)
[ERROR] dependency: tools.vitruv:tools.vitruv.change.interaction.model:jar:3.2.0-SNAPSHOT (compile)
[ERROR] 	Could not find artifact tools.vitruv:tools.vitruv.change.interaction.model:jar:3.2.0-SNAPSHOT in ossrh-snapshots (https://oss.sonatype.org/content/repositories/snapshots)
[ERROR] 	Could not find artifact tools.vitruv:tools.vitruv.change.interaction.model:jar:3.2.0-SNAPSHOT in central-portal-snapshots (https://central.sonatype.com/repository/maven-snapshots/)
[ERROR] dependency: tools.vitruv:tools.vitruv.change.atomic:jar:3.2.0-SNAPSHOT (compile)
[ERROR] 	Could not find artifact tools.vitruv:tools.vitruv.change.atomic:jar:3.2.0-SNAPSHOT in ossrh-snapshots (https://oss.sonatype.org/content/repositories/snapshots)
[ERROR] 	Could not find artifact tools.vitruv:tools.vitruv.change.atomic:jar:3.2.0-SNAPSHOT in central-portal-snapshots (https://central.sonatype.com/repository/maven-snapshots/)
[ERROR] dependency: tools.vitruv:tools.vitruv.change.testutils.integration:jar:3.2.0-SNAPSHOT (compile)
[ERROR] 	Could not find artifact tools.vitruv:tools.vitruv.change.testutils.integration:jar:3.2.0-SNAPSHOT in ossrh-snapshots (https://oss.sonatype.org/content/repositories/snapshots)
[ERROR] 	Could not find artifact tools.vitruv:tools.vitruv.change.testutils.integration:jar:3.2.0-SNAPSHOT in central-portal-snapshots (https://central.sonatype.com/repository/maven-snapshots/)
[ERROR] dependency: tools.vitruv:tools.vitruv.framework.p2wrappers.emfutils:jar:3.2.0-SNAPSHOT (compile)
[ERROR] 	Could not find artifact tools.vitruv:tools.vitruv.framework.p2wrappers.emfutils:jar:3.2.0-SNAPSHOT in ossrh-snapshots (https://oss.sonatype.org/content/repositories/snapshots)
[ERROR] 	Could not find artifact tools.vitruv:tools.vitruv.framework.p2wrappers.emfutils:jar:3.2.0-SNAPSHOT in central-portal-snapshots (https://central.sonatype.com/repository/maven-snapshots/)
[ERROR] 
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :tools.vitruv.cli.cli
```

--- a/cli/pom.xml
+++ b/cli/pom.xml
@@ -76,12 +76,12 @@
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>tools.vitruv.framework.views</artifactId>
-      <version>${project.version}</version>
+      <version>3.2.2</version>
     </dependency>
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>tools.vitruv.framework.vsum</artifactId>
-      <version>${project.version}</version>
+      <version>3.2.2</version>
     </dependency>
 
     <!-- Vitruvius dependencies -->
@@ -135,7 +135,7 @@
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>tools.vitruv.framework.p2wrappers.emfutils</artifactId>
-      <version>${project.version}</version>
+      <version>3.2.2</version>
     </dependency>
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>

--- a/pom.xml
+++ b/pom.xml
@@ -42,9 +42,9 @@
     </modules>
 
     <properties>
-        <vitruv.version>3.2.0-SNAPSHOT</vitruv.version>
-        <vitruv-change.version>3.2.0-SNAPSHOT</vitruv-change.version>
-        <vitruv-dsls.version>3.1.2</vitruv-dsls.version>
+        <vitruv.version>3.2.2</vitruv.version>
+        <vitruv-change.version>3.2.2</vitruv-change.version>
+        <vitruv-dsls.version>3.2.2</vitruv-dsls.version>
         <sonar.host.url>https://sonarcloud.io</sonar.host.url>
         <sonar.organization>vitruv-tools</sonar.organization>
         <sonar.projectKey>vitruv-tools_Vitruv-CLI</sonar.projectKey>

